### ISSUE TITLE
Conflicts raised when moving concepts/views from a folder to another

### ISSUE THREAD

**Issue by jbsarrodie** [archi-contribs/archi-json-plugin#5]
_I'm logging this issue in case someone else face it_

I've recently noticed that when moving a concept or a view from a folder to another, there might be conflicts that should not happen. This is in fact due to a (rather old) regression in the code that store the concepts reference Ids as relative. So when moving something, all relationship or view referencing it are automatically updated which might cause conflicts if at the sametime someone else edited one of those relationships or views.

This will be fixed in an upcomming release soon.

**Comment by jayveeAtWork** [comment]
I've noticed the same issue in the past: when moving e.g. an application component from one folder to another, all the views on which that component is visible need to be updated. If at the same time someone else was working on any of those views for another reason, we get a conflict.

Has this been fixed yet? I noticed on the project page the item "Remove relative URI in XML files (this is the origin of lots of conflicts when moving concepts accross model folders)" is set to "done" but does this only refer to the version which will work with Archi 4?
